src/heros/ItemPrinter.java

```diff
@@ -6,7 +6,7 @@
  * http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
  * 
  * Contributors:
- *     Eric Bodden - initial API and implementation
```

Comment:
As this file was completely written by me, I thought it made more semantic sense to change the attribution for initial API and implementation to me. But if we want to keep Eric as the main contributor of the project that's 100% okay with me.